PositionalGuide/ConfigWindow.cs

```diff
@@ -313,6 +315,7 @@ public class ConfigWindow: Window, IDisposable {
 			this.conf.DrawGuides = drawing;
 			this.conf.LineColours = colours;
 			Plugin.Interface.SavePluginConfig(this.conf);
+			OnSettingsUpdate?.Invoke();
```

Comment:
To fit the style of the repo, instance methods and properties should be qualified with `this.` on access.

Reply:
The reason for this convention choice is that local variable (including method parameters) shadow properties of the same name, which would require using `this.` to differentiate them to the compiler in such cases, and I felt that a uniform style of using `this.` was clearer than using it only in some cases.

Reply:
No idea if you use visual studio. For me all these options are active https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0003-ide0009

but it somehow does not suggest it here but does at other points.